ModelFiles/xml/yeastGEM.xml

```diff
@@ -199809,12 +199839,17 @@
         <notes>
           <body xmlns="http://www.w3.org/1999/xhtml">
             <p>Confidence Level: 0</p>
-            <p>NOTES: added after Gapfilling (PR #185); MNXR102871</p>
+            <p>NOTES: added after Gapfilling (PR #185); rhea:32823; bigg:PItg; unknown transporter</p>
```

Comment:
there are some BiGG ids that can be stored in `rxnBiGGID`

Reply:
mostly addressed in [c82e696](https://github.com/SysBioChalmers/yeast-GEM/pull/226/commits/c82e6962ec020ec14a231bf3de36286c3d1a79d5) (a few are still left but they can be included when moving all notes info to annotation as mentioned in #231)